packaging/helm/skipper-aws/templates/kube-aws-iam-controller/crd.yaml

```diff
@@ -0,0 +1,64 @@
+{{ if .Values.install_kube_aws_iam_controller }}
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
```

Comment:
When using CRDs with Helm make sure you're aware of the limitations: https://helm.sh/docs/topics/chart_best_practices/custom_resource_definitions/

Basically, updates / deleted of CRDs isn't supported and to ensure CRDs are installed first (before any resources that rely on them) they either need to go in a directory called `crds` or manually applied to the cluster prior to the helm install.

Reply:
will move them to crds/